### ISSUE TITLE
Modifying the loss and underlying models

### ISSUE THREAD

**Issue by nilabjodey** [zhanghang1989/PyTorch-Encoding#236]
How can I train a new model while modifying the loss function used in the encnet as well as the deeplab / fcn.. model cross-entropy loss ? 
Also, how can I remove encnet from the structure and do an ablation study?

**Comment by StacyYang** [comment]
Please read the docs on PyTorch for customizing models.
